[
{"message":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","level":"FATAL","type":"PLAYGROUND","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"file:///home/runner/work/playground-tester/playground-tester/node_modules/@wp-playground/cli/index.js:237\n        throw new Error(y, { cause: u });\n              ^\n","log":"/tmp/tmp.wIQr78Rf6P"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:19 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:20 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:28 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:29 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:37 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","level":"DEPRECATED","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:38 UTC] PHP Deprecated:  Required parameter $tag follows optional parameter $content in /wordpress/wp-content/plugins/oberon-wpaddin/owpa_plugin.php on line 103","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"openssl_pkey_export(): cannot get key from parameter 1 in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 376","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:48 UTC] PHP Warning:  openssl_pkey_export(): cannot get key from parameter 1 in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 376","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"openssl_pkey_get_details() expects parameter 1 to be resource, bool given in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 381","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:48 UTC] PHP Warning:  openssl_pkey_get_details() expects parameter 1 to be resource, bool given in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 381","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Trying to access array offset on value of type null in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 382","level":"NOTICE","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:48 UTC] PHP Notice:  Trying to access array offset on value of type null in /wordpress/wp-content/plugins/oauth2-provider/includes/functions.php on line 382","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(358): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(359): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(361): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(362): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(363): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM wp_oauth_access_tokens</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(375): wpdb->get_results('SHOW KEYS FROM ...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#7 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#8 {main}</pre>\n for query SHOW KEYS FROM wp_oauth_access_tokens made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(383): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_access_tokens, `AP_GENERATED`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_access_tokens ADD `ap_generated`   text  \t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(391): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE`, varchar(1000)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(401): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:26:58 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE_METHOD`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(411): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(358): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY refresh_token VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(359): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_refresh_tokens MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(361): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_public_keys MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(362): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_jwt MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(363): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes MODIFY client_id VARCHAR(191); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown show type: KEYS FROM.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW KEYS FROM wp_oauth_access_tokens</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown show type: KEYS FROM.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3146): WP_SQLite_DB->query('SHOW KEYS FROM ...')\n#2 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(375): wpdb->get_results('SHOW KEYS FROM ...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#7 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#8 {main}</pre>\n for query SHOW KEYS FROM wp_oauth_access_tokens made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Unknown operation: MODIFY.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000);</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Unknown operation: MODIFY.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(383): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens MODIFY access_token VARCHAR(1000); made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_access_tokens, `AP_GENERATED`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_access_tokens ADD `ap_generated`   text  \t | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: ap_generated.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(391): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_access_tokens ADD `ap_generated` VARCHAR( 32 ) AFTER \t`scope` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE`, varchar(1000)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(401): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge` VARCHAR( 1000 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:27:09 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token`</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: INSERT INTO _mysql_data_types_cache (`table`, `column_or_index`, `mysql_type`)\n\t\t\t\tVALUES (:table, :column, :datatype)\n\t\t\t\tON CONFLICT(`table`, `column_or_index`) DO UPDATE SET `mysql_type` = :datatype\n\t\t\t | parameters: wp_oauth_authorization_codes, `CODE_CHALLENGE_METHOD`, varchar(32)</li>\n<li>Executing: ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method`   text   | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 duplicate column name: code_challenge_method.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/oauth2-provider/wp-oauth-main.php(411): WP_SQLite_DB->query('ALTER TABLE wp_...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): WO_Server->upgrade(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters(NULL, Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_oauth2...', false)\n#6 /internal/eval.php(20): activate_plugin('oauth2-provider...')\n#7 {main}</pre>\n for query ALTER TABLE wp_oauth_authorization_codes ADD `code_challenge_method` VARCHAR( 32 ) AFTER `id_token` made by activate_plugin, do_action('activate_oauth2-provider/wp-oauth-server.php'), WP_Hook->do_action, WP_Hook->apply_filters, WO_Server->upgrade, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:28:42 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133\nStack trace:\n#0 /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_options.php(148): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('occupancyplan/o...')\n#3 /internal/eval.php(20): activate_plugin('occupancyplan/o...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php on line 133","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:28:50 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133\nStack trace:\n#0 /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_options.php(148): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('occupancyplan/o...')\n#3 /internal/eval.php(20): activate_plugin('occupancyplan/o...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php on line 133","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133","level":"FATAL","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:28:56 UTC] PHP Fatal error:  Uncaught Error: Call to undefined function create_function() in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php:133\nStack trace:\n#0 /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_options.php(148): require_once()\n#1 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#2 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('occupancyplan/o...')\n#3 /internal/eval.php(20): activate_plugin('occupancyplan/o...')\n#4 {main}\n  thrown in /wordpress/wp-content/plugins/occupancyplan/occupancy_plan_widget.php on line 133","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Undefined array key 3 in /internal/shared/sqlite-database-integration/wp-includes/parser/class-wp-parser.php on line 55","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:03 UTC] PHP Warning:  Undefined array key 3 in /internal/shared/sqlite-database-integration/wp-includes/parser/class-wp-parser.php on line 55","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Attempt to read property \"id\" on null in /internal/shared/sqlite-database-integration/wp-includes/parser/class-wp-parser.php on line 55","level":"WARNING","type":"PHP","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:03 UTC] PHP Warning:  Attempt to read property \"id\" on null in /internal/shared/sqlite-database-integration/wp-includes/parser/class-wp-parser.php on line 55","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_ochregeo_geocoder_cache.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:12 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DELETE FROM wp_ochregeo_geocoder_cache WHERE tt < 1748470812</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DELETE FROM wp_ochregeo_geocoder_cache WHERE tt &lt; 1748470812 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_ochregeo_geocoder_cache.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(435): WP_SQLite_DB->query('DELETE FROM wp_...')\n#2 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(60): OCHRELABS_WP_Geolocation->geocoder_cache_gc()\n#3 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(516): OCHRELABS_WP_Geolocation->__construct()\n#4 /wordpress/wp-admin/includes/plugin.php(2387): include_once('/wordpress/wp-c...')\n#5 /wordpress/wp-admin/includes/plugin.php(673): plugin_sandbox_scrape('ochre-w3c-geolo...')\n#6 /internal/eval.php(20): activate_plugin('ochre-w3c-geolo...')\n#7 {main}</pre>\n for query DELETE FROM wp_ochregeo_geocoder_cache WHERE tt < 1748470812 made by activate_plugin, plugin_sandbox_scrape, include_once('/plugins/ochre-w3c-geolocation-services/ochre-geo.php'), OCHRELABS_WP_Geolocation->__construct, OCHRELABS_WP_Geolocation->geocoder_cache_gc, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"Array\n(\n    [query] => SHOW TABLES LIKE 'wp_ochregeo_geocoder_cache\n    [message] => Ending quote ' was expected.\n    [str] => \n    [position] => 44\n    [code] => 0\n)\n.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:12 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>SHOW TABLES LIKE 'wp_ochregeo_geocoder_cache</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: Array\n(\n    [query] => SHOW TABLES LIKE 'wp_ochregeo_geocoder_cache\n    [message] => Ending quote ' was expected.\n    [str] => \n    [position] => 44\n    [code] => 0\n)\n.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-includes/class-wpdb.php(3031): WP_SQLite_DB->query('SHOW TABLES LIK...')\n#2 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(388): wpdb->get_var('SHOW TABLES LIK...')\n#3 /wordpress/wp-includes/class-wp-hook.php(324): OCHRELABS_WP_Geolocation->activate(false)\n#4 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#5 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#6 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ochre-...', false)\n#7 /internal/eval.php(20): activate_plugin('ochre-w3c-geolo...')\n#8 {main}</pre>\n for query SHOW TABLES LIKE 'wp_ochregeo_geocoder_cache made by activate_plugin, do_action('activate_ochre-w3c-geolocation-services/ochre-geo.php'), WP_Hook->do_action, WP_Hook->apply_filters, OCHRELABS_WP_Geolocation->activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"MySQL field type cannot be translated to SQLite: serial.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:12 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>CREATE TABLE wp_ochregeo_geocoder_cache ( \n        id serial,\n        la decimal(10,5),\n        ll decimal(10,5),\n        tt bigint(11),\n        dat text,\n        primary key(id),\n        index(la),\n        index(ll),\n        index(tt)\n        )</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: MySQL field type cannot be translated to SQLite: serial.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(389): WP_SQLite_DB->query('CREATE TABLE wp...')\n#2 /wordpress/wp-includes/class-wp-hook.php(324): OCHRELABS_WP_Geolocation->activate(false)\n#3 /wordpress/wp-includes/class-wp-hook.php(348): WP_Hook->apply_filters('', Array)\n#4 /wordpress/wp-includes/plugin.php(517): WP_Hook->do_action(Array)\n#5 /wordpress/wp-admin/includes/plugin.php(703): do_action('activate_ochre-...', false)\n#6 /internal/eval.php(20): activate_plugin('ochre-w3c-geolo...')\n#7 {main}</pre>\n for query CREATE TABLE wp_ochregeo_geocoder_cache ( \n        id serial,\n        la decimal(10,5),\n        ll decimal(10,5),\n        tt bigint(11),\n        dat text,\n        primary key(id),\n        index(la),\n        index(ll),\n        index(tt)\n        ) made by activate_plugin, do_action('activate_ochre-w3c-geolocation-services/ochre-geo.php'), WP_Hook->do_action, WP_Hook->apply_filters, OCHRELABS_WP_Geolocation->activate, WP_SQLite_DB->query, WP_SQLite_DB->print_error","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
,
{"message":"SQLSTATE[HY000]: General error: 1 no such table: wp_ochregeo_geocoder_cache.","level":"FATAL","type":"SQL","test":"asyncify-boot","plugin":"obm-hit-counter-bangla","details":"[28-May-2025 22:30:22 UTC] WordPress database error <div style=\"clear:both\">&nbsp;</div>\n<div class=\"queries\" style=\"clear:both;margin-bottom:2px;border:red dotted thin;\">\n<p>MySQL query:</p>\n<p>DELETE FROM wp_ochregeo_geocoder_cache WHERE tt < 1748470822</p>\n<p>Queries made or created this session were:</p>\n<ol>\n<li>Executing: BEGIN | (no parameters)</li>\n<li>Executing: DELETE FROM wp_ochregeo_geocoder_cache WHERE tt &lt; 1748470822 | (no parameters)</li>\n<li>Executing: ROLLBACK | (no parameters)</li>\n</ol>\n</div>\n<div style=\"clear:both;margin-bottom:2px;border:red dotted thin;\" class=\"error_message\" style=\"border-bottom:dotted blue thin;\">\nError occurred at line 4117 in Function <code>handle_error</code>. Error message was: SQLSTATE[HY000]: General error: 1 no such table: wp_ochregeo_geocoder_cache.\n</div>\n<p>Backtrace:</p>\n<pre>#0 /internal/shared/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-db.php(287): WP_SQLite_Translator->get_error_message()\n#1 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(435): WP_SQLite_DB->query('DELETE FROM wp_...')\n#2 /wordpress/wp-content/plugins/ochre-w3c-geolocation-services/ochre-geo.php(60)\n","log":"/logs/plugins/o/obm-hit-counter-bangla/error.json"}
]
